docs/snippets/compare_snippet_output.py

```diff
@@ -56,6 +58,15 @@ def extra_args(self) -> list[str]:
     def output_path(self, language: str) -> str:
         return f"docs/snippets/all/{self.subdir}/{self.name}_{language}.rrd"
 
+    def backwards_compatibility_path(self) -> Path:
+        """
+        Files checked in to CI that tests backwards compatibility.
+
+        We use this path as a source for comparison ("are old files correctly migrated?")
+        and as a destination when these files need updating using --write-missing-backward-assets.
+        """
+        return Path(f"tests/assets/rrd/snippets/{self.subdir}/{self.name}.rrd")
```

Comment:
As discussed internally, maybe add version to the path here.

Reply:
Added to the TODO of https://github.com/rerun-io/rerun/issues/9110